src/core/live-announcer/live-announcer.ts

```diff
@@ -0,0 +1,52 @@
+import {Injectable} from 'angular2/core';
+import {DOM} from '../../core/platform/dom/dom_adapter';
+
+@Injectable()
+export class MdLiveAnnouncer {
+
+  private announcerEl: Element;
+
+  constructor() {
+    this.announcerEl = this._getAnnouncerElement();
+  }
+
+  /**
+   * @param message Message to be announced to the screenreader
+   * @param politeness The politeness of the announcer element.
+   */
+  announce(message: string, politeness = 'polite'): void {
+    this.announcerEl.textContent = '';
+
+    this.announcerEl.setAttribute('aria-live', politeness);
+
+    // This 100ms timeout is necessary for some browser + screen-reader combinations:
+    // - Both JAWS and NVDA over IE11 will not announce anything without a non-zero timeout.
+    // - With Chrome and IE11 with NVDA or JAWS, a repeated (identical) message won't be read a
+    //   second time without clearing and then using a non-zero delay.
+    // (using JAWS 17 at time of this writing).
+    setTimeout(() => this.announcerEl.textContent = message, 100);
+  }
+
+  private _getAnnouncerElement(): Element {
+    let documentBody = DOM.getGlobalEventTarget('body');
```

Comment:
Since we decided to drop TS support yesterday, you can simply use `document.createElement`, etc.

Instead of looking for an existing announcer element, the service should just always create one.